test/app.test.js

```diff
@@ -628,7 +628,7 @@ describe('app', function() {
       var Foo = app.models.foo;
       var f = new Foo();
 
-      assert(f instanceof loopback.Model);
+      assert(f instanceof app.registry.getModel('Model'));
```

Comment:
I'd like to avoid any `regsitry` "special" properties like this. Model will always be available via `getModel('Model')`.